tensorflow_addons/layers/wrappers.py

```diff
@@ -92,34 +93,36 @@ def build(self, input_shape):
         self.kernel_norm_axes = list(range(kernel.shape.rank - 1))
 
         self.g = self.add_weight(
-            name='g',
+            name="g",
             shape=(self.layer_depth,),
-            initializer='ones',
+            initializer="ones",
             dtype=kernel.dtype,
-            trainable=True)
+            trainable=True,
+        )
         self.v = kernel
 
         self._initialized = self.add_weight(
-            name='initialized',
+            name="initialized",
             shape=None,
-            initializer='zeros',
+            initializer="zeros",
             dtype=tf.dtypes.bool,
-            trainable=False)
+            trainable=False,
+        )
 
         if self.data_init:
             # Used for data initialization in self._data_dep_init.
-            with tf.name_scope('data_dep_init'):
+            with tf.name_scope("data_dep_init"):
                 layer_config = tf.keras.layers.serialize(self.layer)
-                layer_config['config']['trainable'] = False
-                self._naked_clone_layer = tf.keras.layers.deserialize(
-                    layer_config)
+                layer_config["config"]["trainable"] = False
+                self._naked_clone_layer = tf.keras.layers.deserialize(layer_config)
                 self._naked_clone_layer.build(input_shape)
                 self._naked_clone_layer.set_weights(self.layer.get_weights())
                 if not self.is_rnn:
                     self._naked_clone_layer.activation = None
 
         self.built = True
 
+    @tf.function(autograph=False)
```

Comment:
I'm guessing this fixes the issue. Can you add a test to check for this?

Reply:
Sorry, I don't know how to write such test case